st2common/st2common/constants/sensors.py

```diff
@@ -15,3 +15,8 @@
 
 # Minimum poll interval for a sensor
 MINIMUM_POLL_INTERVAL = 4
+
+# keys for PARTITION loaders
+DEFAULT_PARTITION_LOADER = 'default'
```

Comment:
Sure. It looks a lot smarter although not sure what I would do with that URI structure.